src/compiler/target/mod.rs

```diff
@@ -13,12 +15,14 @@ pub use tokenize::{
 #[derive(Clone)]
 pub enum Lang {
   JS,
```

Comment:
Can you add this here?
```suggestion
  // Currently dead because we do not yet have a working CLI with a JS option
  // TODO: fix this
  #[allow(dead_code)]
  JS,
```